app/services/savings_pass_fail_service.rb

```diff
@@ -0,0 +1,67 @@
+class SavingsPassFailService
```

Comment:
You, sir, are correct!  Two many similarly named fields!   Yes, I was going to add a setting for min and max values and import them... 
On my todo list now! :)